web/src/reducers/activities.js

```diff
@@ -291,23 +292,22 @@ const reducer = (state = initialState, action) => {
 
       // if switching to hourly, clear out yearly totals
       // if switch to yearly, clear out hourly numbers
-      const contractorResources = contractorsOld.map(
-        contractor =>
-          contractor.key !== contractorKey
-            ? contractor
-            : {
-                ...contractor,
-                ...(useHourly && {
-                  years: arrToObj(years, contractorDefaultYear())
-                }),
-                hourly: {
-                  ...contractor.hourly,
-                  useHourly,
-                  ...(!useHourly && {
-                    data: arrToObj(years, contractorDefaultHourly())
-                  })
-                }
+      const contractorResources = contractorsOld.map(contractor =>
+        contractor.key !== contractorKey
+          ? contractor
+          : {
+              ...contractor,
+              ...(useHourly && {
+                years: arrToObj(years, contractorDefaultYear())
+              }),
+              hourly: {
+                ...contractor.hourly,
+                useHourly,
+                ...(!useHourly && {
+                  data: arrToObj(years, contractorDefaultHourly())
```

Comment:
lolz, I spent about 20 minutes trying to parse this before I realized the actual change was just moving the arrow function up a line. 🤦‍♀️